README.md

```diff
@@ -1 +1,60 @@
 # st21-chestX
+<b>ChestX</b> is a <b>TechLabs Berlin</b> project that aims to detect COVID-19 infection by processing chest X-ray images. To support medical decission making, the ChestX team wants to applying AI technics for object detection & classification, as well as data science methodes. Metadata provieded alongside the X-ray images can be analysed to providing doctors with more meaningful diagnostic assistance.
+
```

Comment:
<b>ChestX</b> is a <b>TechLabs Berlin</b> project that aims to detect COVID-19 infection by processing chest X-ray images. To support medical decision making, the ChestX team wants to apply AI techniques for object detection & classification, as well as data science methods. Metadata provided alongside the X-ray images can be analysed to provide doctors with more meaningful diagnostic assistance.

Reply:
Thanks! That's geat. I corrected the two typos and updated the file. Would merge this now after a second review to approve the changes.